Formula/kontena-cli.rb

```diff
@@ -0,0 +1,68 @@
+class KontenaCli < Formula
+  desc "Command-line client for Kontena container & microservices platform"
+  homepage "https://kontena.io/"
+  url "https://github.com/kontena/kontena.git", :tag => "v1.3.4"
```

Comment:
needs to specify the revision too. see the `osquery` formula for how it should be formatted